[2024-05-19T00:00:26.197+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-19T00:00:28.312+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Real_Estate_Webscraping_ETL.Process_and_Send_Data_to_DB scheduled__2024-05-18T12:05:00+00:00 [queued]>
[2024-05-19T00:00:28.517+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Real_Estate_Webscraping_ETL.Process_and_Send_Data_to_DB scheduled__2024-05-18T12:05:00+00:00 [queued]>
[2024-05-19T00:00:28.524+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 6
[2024-05-19T00:00:28.922+0000] {taskinstance.py:2330} INFO - Executing <Task(SparkSubmitOperator): Process_and_Send_Data_to_DB> on 2024-05-18 12:05:00+00:00
[2024-05-19T00:00:34.413+0000] {standard_task_runner.py:63} INFO - Started process 709 to run task
[2024-05-19T00:00:34.604+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Real_Estate_Webscraping_ETL', 'Process_and_Send_Data_to_DB', 'scheduled__2024-05-18T12:05:00+00:00', '--job-id', '412', '--raw', '--subdir', 'DAGS_FOLDER/webscrapping.py', '--cfg-path', '/tmp/tmpbxqdr2fe']
[2024-05-19T00:00:35.169+0000] {standard_task_runner.py:91} INFO - Job 412: Subtask Process_and_Send_Data_to_DB
[2024-05-19T00:00:47.214+0000] {task_command.py:426} INFO - Running <TaskInstance: Real_Estate_Webscraping_ETL.Process_and_Send_Data_to_DB scheduled__2024-05-18T12:05:00+00:00 [running]> on host 7879427f0394
[2024-05-19T00:00:50.257+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='svc' AIRFLOW_CTX_DAG_ID='Real_Estate_Webscraping_ETL' AIRFLOW_CTX_TASK_ID='Process_and_Send_Data_to_DB' AIRFLOW_CTX_EXECUTION_DATE='2024-05-18T12:05:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-18T12:05:00+00:00'
[2024-05-19T00:00:50.653+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-19T00:00:51.721+0000] {base.py:84} INFO - Using connection ID 'spark_default' for task execution.
[2024-05-19T00:00:51.754+0000] {spark_submit.py:401} INFO - Spark-Submit cmd: spark-submit --master spark://spark-master:7077 --packages org.apache.spark:spark-sql-kafka-0-10_2.12:3.5.1,com.datastax.spark:spark-cassandra-connector_2.12:3.5.0,org.apache.kafka:kafka-clients:3.5.1,commons-logging:commons-logging:1.2 --name arrow-spark --deploy-mode client dags/includes/jobs/spark_consumer.py
[2024-05-19T00:00:53.881+0000] {spark_submit.py:571} INFO - /home/***/.local/lib/python3.11/site-packages/pyspark/bin/load-spark-env.sh: line 68: ps: command not found
[2024-05-19T00:01:31.970+0000] {local_task_job_runner.py:124} ERROR - Received SIGTERM. Terminating subprocesses
[2024-05-19T00:01:32.447+0000] {process_utils.py:132} INFO - Sending 15 to group 709. PIDs of all processes in the group: [736, 709]
[2024-05-19T00:01:32.525+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 709
[2024-05-19T00:01:33.515+0000] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-05-19T00:01:33.932+0000] {spark_submit.py:697} INFO - Sending kill signal to spark-submit
[2024-05-19T00:01:34.002+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-19T00:01:38.785+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 490, in submit
    self._process_spark_submit_log(iter(self._submit_sp.stdout))  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 539, in _process_spark_submit_log
    for line in itr:
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 2613, in signal_handler
    raise AirflowTaskTerminated("Task received SIGTERM signal")
airflow.exceptions.AirflowTaskTerminated: Task received SIGTERM signal
[2024-05-19T00:01:39.533+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=Real_Estate_Webscraping_ETL, task_id=Process_and_Send_Data_to_DB, run_id=scheduled__2024-05-18T12:05:00+00:00, execution_date=20240518T120500, start_date=20240519T000028, end_date=20240519T000139
[2024-05-19T00:01:40.251+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=736, status='terminated', started='00:00:51') (736) terminated with exit code None
[2024-05-19T00:01:40.407+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=709, status='terminated', exitcode=2, started='00:00:33') (709) terminated with exit code 2
[2024-05-19T00:01:40.442+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 143
